src/main/java/seedu/address/model/person/PriorityValue.java

```diff
@@ -9,42 +9,33 @@
  */
 public class PriorityValue {
 
-    private static final String SPECIAL_CHARACTERS = "!#$%&'*+/=?`{|}~^.-";
-    public static final String MESSAGE_PRIORITYVALUE_CONSTRAINTS = "Emails should be of the format local-part@domain "
-            + "and adhere to the following constraints:\n"
-            + "1. The local-part should only contain alphanumeric characters and these special characters, excluding "
-            + "the parentheses, (" + SPECIAL_CHARACTERS + ") .\n"
-            + "2. This is followed by a '@' and then a domain name. "
-            + "The domain name must:\n"
-            + "    - be at least 2 characters long\n"
-            + "    - start and end with alphanumeric characters\n"
-            + "    - consist of alphanumeric characters, a period or a hyphen for the characters in between, if any.";
+    //private static final String SPECIAL_CHARACTERS = "!#$%&'*+/=?`{|}~^.-";
+    public static final String MESSAGE_PRIORITY_VALUE_CONSTRAINTS = "Priority value should be a positive integer. ";
```

Comment:
is the whitespace after the full stop intentional?

Reply:
Er no it's not, will fix in a future PR